<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Turbo Rascal Syntax error, &quot;;&quot; expected but &quot;BEGIN&quot; Version @version</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1000,0">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnNewProject">
              <property name="text">
               <string>New Project</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnProjectSettings">
              <property name="text">
               <string>Project Settings</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="chkShowAllFiles">
              <property name="text">
               <string>Show all</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="qsplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QSplitter" name="psplitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QTreeView" name="treeFiles">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QTreeWidget" name="treeSymbols">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Filter:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leFilterSymbols"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="1" column="1">
              <widget class="QPushButton" name="btnMemoryAnalyze">
               <property name="toolTip">
                <string>Opens up a memory map analyzer window</string>
               </property>
               <property name="text">
                <string>Memory Map (c-u)</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="btnBuildAll">
               <property name="toolTip">
                <string>Builds all files according to the build instruction script. Check the &quot;project settings&quot; for details.</string>
               </property>
               <property name="text">
                <string>Build All</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnBuild_3">
               <property name="text">
                <string>Build (c-b)</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btnShowcases">
               <property name="text">
                <string>TRSE Showcases</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QPushButton" name="btnClearProject">
               <property name="toolTip">
                <string>Deletes all .asm .sym .prg files in the current project</string>
               </property>
               <property name="text">
                <string>Clear project</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="btnBuild_4">
               <property name="toolTip">
                <string>Runs the current program. </string>
               </property>
               <property name="text">
                <string>Run (c-r)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="btnSave_3">
               <property name="text">
                <string>Save (c-s)</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QPushButton" name="btnProjectDir">
               <property name="toolTip">
                <string>Opens up an explorer window to the current project folder</string>
               </property>
               <property name="text">
                <string>Project directory</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btnSizeAnalyser">
               <property name="toolTip">
                <string>Opens up a memory map analyzer window</string>
               </property>
               <property name="text">
                <string>Size analyser (c-y)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
            <item>
             <layout class="QGridLayout" name="gridLayout_5" rowstretch="5,0">
              <item row="0" column="0">
               <widget class="QLabel" name="lblCommodoreImage">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="resources.qrc">:/resources/images/C64.png</pixmap>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblSystemName">
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblSave">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>48</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>48</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>63</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/resources/images/tape.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblBuild">
            <property name="maximumSize">
             <size>
              <width>48</width>
              <height>48</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resources.qrc">:/resources/images/hammer.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>49</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabMain">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="tab_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <attribute name="title">
         <string>Turbo Rascal Solution</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0" columnstretch="0">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_3" columnstretch="5,12,3">
             <item row="0" column="1">
              <widget class="LGLSlideshow" name="glslider" native="true"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>128</width>
                 <height>128</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>128</width>
                 <height>128</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="resources.qrc">:/resources/images/trse_icon.png</pixmap>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="LGLShowcases" name="glShowcase" native="true">
               <property name="toolTip">
                <string>Click to open URL to production</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab">
            <attribute name="title">
             <string>Sample projects and tutorials</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,12">
               <item row="1" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
                 <item>
                  <widget class="QComboBox" name="cmbSelectSystemRecent"/>
                 </item>
                 <item>
                  <widget class="QListWidget" name="lstRecentProjects">
                   <property name="font">
                    <font>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="1">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,0,0" columnstretch="0">
                   <item row="2" column="0">
                    <widget class="QScrollArea" name="scrollArea">
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <widget class="QWidget" name="scrollAreaWidgetContents">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>553</width>
                        <height>358</height>
                       </rect>
                      </property>
                     </widget>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
                     <item>
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>Select system:</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cmbSelectSystem"/>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="NewsAndUpdates">
            <attribute name="title">
             <string>News and updates</string>
            </attribute>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="0" column="0">
              <widget class="QTextBrowser" name="txtNews">
               <property name="html">
                <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;News and updates from TRSE!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>&amp;New file</string>
     </property>
     <addaction name="actionRas_source_file"/>
     <addaction name="action_TRU_Unit_source_file"/>
     <addaction name="action_include_source_file"/>
     <addaction name="actionImage"/>
     <addaction name="actionPaw_packed_resource_file"/>
     <addaction name="actionFjong_Raymarcher_document"/>
     <addaction name="actionTRSE_Tracker_File_trt"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionNew_project"/>
    <addaction name="actionOpen_project"/>
    <addaction name="actionOpen_project_location"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="action_Rename_current_file"/>
    <addaction name="actionDelete_file"/>
    <addaction name="actionClose_current_project"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionSearch"/>
    <addaction name="actionSearch_in_symbols_c_g"/>
    <addaction name="actionLook_up_symbol_F2"/>
    <addaction name="actionLook_up_assembly_line_under_cursor_F3"/>
    <addaction name="action_Comment_Uncomment"/>
    <addaction name="actionAuto_ident"/>
    <addaction name="actionProject_Settings"/>
    <addaction name="actionFind_file_c_s"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionTRSE_Settings"/>
    <addaction name="actionWavefront_obj_to_amiga_converter"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp_F1"/>
    <addaction name="actionTRSE_Showcases"/>
    <addaction name="actionDonate"/>
    <addaction name="actionAbout"/>
    <addaction name="actionView_changelog"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>&amp;Window</string>
    </property>
    <addaction name="actionClose_current"/>
    <addaction name="actionClose_all"/>
   </widget>
   <widget class="QMenu" name="menuProject">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionBuild_C_b"/>
    <addaction name="actionBuild_current_File_only_Shift_C_b"/>
    <addaction name="actionBuild_All"/>
    <addaction name="action_Run"/>
    <addaction name="action_Memory_map_C_u"/>
    <addaction name="action_Project_Settings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProject"/>
   <addaction name="menuTools"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionRas_source_file">
   <property name="text">
    <string>Ras source file (.ras)</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image/Charset/Sprite (.flf)</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>&amp;Search (c-f)</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </action>
  <action name="actionAuto_ident">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autoformat (c-j)</string>
   </property>
  </action>
  <action name="actionDelete_file">
   <property name="text">
    <string>Delete file</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionTRSE_Settings">
   <property name="text">
    <string>&amp;TRSE Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_project">
   <property name="text">
    <string>New project</string>
   </property>
  </action>
  <action name="actionClose_current">
   <property name="text">
    <string>&amp;Close current (c-w)</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close &amp;All</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>&amp;Open project</string>
   </property>
  </action>
  <action name="actionClose_current_project">
   <property name="text">
    <string>&amp;Close current project</string>
   </property>
  </action>
  <action name="actionHelp_F1">
   <property name="text">
    <string>Help F1</string>
   </property>
  </action>
  <action name="actionMovie_Creator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Movie Creator</string>
   </property>
  </action>
  <action name="actionCheck_for_new_version">
   <property name="text">
    <string>Check for new version</string>
   </property>
  </action>
  <action name="actionPaw_packed_resource_file">
   <property name="text">
    <string>Paw packed resource file (.paw)</string>
   </property>
  </action>
  <action name="actionProject_Settings">
   <property name="text">
    <string>&amp;Project Settings</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionFind_file_c_s">
   <property name="text">
    <string>&amp;Find file (c-l)</string>
   </property>
  </action>
  <action name="actionEffects">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Effects</string>
   </property>
  </action>
  <action name="actionFjong_Raymarcher_document">
   <property name="text">
    <string>Fjong Raymarcher document (.fjo)</string>
   </property>
  </action>
  <action name="actionWavefront_obj_to_amiga_converter">
   <property name="text">
    <string>Wavefront .obj converter</string>
   </property>
  </action>
  <action name="actionBuild_C_b">
   <property name="text">
    <string>&amp;Build (C-b)</string>
   </property>
  </action>
  <action name="actionBuild_All">
   <property name="text">
    <string>Build &amp;All</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run (C-r)</string>
   </property>
  </action>
  <action name="action_Memory_map_C_u">
   <property name="text">
    <string>&amp;Memory map (C-u)</string>
   </property>
  </action>
  <action name="action_Project_Settings">
   <property name="text">
    <string>&amp;Project Settings</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionOpen_project_location">
   <property name="text">
    <string>O&amp;pen project location</string>
   </property>
  </action>
  <action name="actionSearch_in_symbols_c_g">
   <property name="text">
    <string>Search in symbols (c-&amp;g)</string>
   </property>
  </action>
  <action name="actionLook_up_symbol_F2">
   <property name="text">
    <string>Look up sy&amp;mbol under cursor (F2)</string>
   </property>
  </action>
  <action name="actionBuild_current_File_only_Shift_C_b">
   <property name="text">
    <string>Build current &amp;File only (Shift-C-b)</string>
   </property>
  </action>
  <action name="action_TRU_Unit_source_file">
   <property name="text">
    <string>&amp;TRU Unit source file (.tru)</string>
   </property>
  </action>
  <action name="actionTRSE_Showcases">
   <property name="text">
    <string>TRSE Showcases</string>
   </property>
  </action>
  <action name="actionView_changelog">
   <property name="text">
    <string>View changelog</string>
   </property>
  </action>
  <action name="action_Comment_Uncomment">
   <property name="text">
    <string>&amp;Comment/Uncomment (c-e)</string>
   </property>
  </action>
  <action name="actionLook_up_assembly_line_under_cursor_F3">
   <property name="text">
    <string>Look up assembly line under cursor (F3)</string>
   </property>
  </action>
  <action name="action_Rename_current_file">
   <property name="text">
    <string>&amp;Rename current file</string>
   </property>
  </action>
  <action name="actionTRSE_Tracker_File_trt">
   <property name="text">
    <string>TRSE Music Tracker File (.trt)</string>
   </property>
  </action>
  <action name="action_include_source_file">
   <property name="text">
    <string>Include file (.inc)</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>S&amp;ave All</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LGLSlideshow</class>
   <extends>QWidget</extends>
   <header>source/LeLib/lglslideshow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LGLShowcases</class>
   <extends>QWidget</extends>
   <header location="global">source/LeLib/lglshowcases.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
